graphdb/run_graphdb.sh

```diff
@@ -4,7 +4,7 @@
 # It reads the username, password, and database name from the environment variables.
 export POD_CONTAINER="${POD_CONTAINER:-docker}"
 export NEO4J_USERNAME="${NEO4J_USERNAME:-neo4j}"
-export NEO4J_PASSWORD="${NEO4J_PASSWORD:-password}"
+export NEO4J_PASSWORD="${NEO4J_PASSWORD:-neo4jpassword}"
 export NEO4J_DATABASE="${NEO4J_DATABASE:-wise}"
```

Comment:
For NEO4J_DATABASE, the default should be "neo4j"

Reply:
Fixed